### PR TITLE
(SIMP-5335) Wrong GitLab env var for Puppet 5

### PR DESCRIPTION
**Pull request by lnemsick-simp**
- Update badges and contribution guide URL in README.rst
- Fix Puppet collection env variable in .gitlab-ci.yml
- Fix incorrect use of BEAKER_PUPPET_ENVIRONMENT/puppet_environment
- Update to onyxpoint OEL boxes in acceptance tests
- Update static assets

SIMP-5335 #comment pupmod-simp-simp_elasticsearch
SIMP-5430 #comment pupmod-simp-simp_elasticsearch
SIMP-4964 #comment pupmod-simp-simp_elasticsearch

### CODE REVIEW

**Comment by lnemsick-simp** [comment]
:heavy_check_mark: https://gitlab.com/simp/pupmod-simp-simp_elasticsearch/pipelines/34732383

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,15 +1,18 @@
 # The testing matrix considers ruby/puppet versions supported by SIMP and PE:
 #
-# https://puppet.com/docs/pe/2017.3/overview/component_versions_in_recent_pe_releases.html
+# https://puppet.com/docs/pe/latest/overview/component_versions_in_recent_pe_releases.html
```
Comment (op-ct) [inline .gitlab-ci.yml]: A link to the documentation under `/latest/` will not necessarily contain this data in the future. 

Data about historical releases is periodically removed from newer versions of the documentation, particularly on the "component releases" page (note that the versions start with 2016.4, the oldest LTS).  Documentation URLs are also occasionally restructured from version to version.  

We've been linking to a specific version of the documentation to ensure it has the correct information. 

```suggestion
# https://puppet.com/docs/pe/2018.1/component_versions_in_recent_pe_releases.html
```

**Review by op-ct — DISMISSED** [review]

**Comment by lnemsick-simp** [comment]
https://gitlab.com/simp/pupmod-simp-simp_elasticsearch/pipelines/34991327

**Review by trevor-vaughan — APPROVED** [review]
All tests are passing in GitLab

**Pull request merged**